client/package.json

```diff
@@ -4,10 +4,11 @@
 	"private": true,
 	"dependencies": {
 		"@formatjs/intl": "^2.10.4",
-		"@make-software/csprclick-core-client": "^1.9.4",
-		"@make-software/csprclick-core-types": "^1.9.4",
-		 "@make-software/csprclick-ui": "^1.9.0",
+		"@make-software/csprclick-core-client": "^1.10.0",
+		"@make-software/csprclick-core-types": "^1.10.0",
+		 "@make-software/csprclick-ui": "^1.10.0",
 		"@metamask/safe-event-emitter": "^2.0.0",
+		"@make-software/cspr-design": "github:make-software/cspr-design#a53a7d2a31fc8a5e0b3dd9272a609422c3d4bedc",
```

Comment:
Let's release a patch version if need to link to some specific changes in the library